tableau-specialist/1.0/modules/ROOT/pages/tableau-specialist-connector-reference.adoc

```diff
@@ -0,0 +1,1255 @@
+
+= Tableau Specialist Connector 1.0 Reference
+
+
+
+== Configurations
+---
+[[TableauHyperConfig]]
+=== Hyper configuration
+
+
+This configuration provides users access to the HyperAPI Java library developed by Tableau. Users can create, update, delete, and modify hyper files locally.
```

Comment:
What is the difference between update and modify here? Just asking so you can double-check that this is correct.

Reply:
The official Tableau site doesn't even have modify as an option, so you're right. I guess they meant modify as update so lemme remove that